components/Common/LanguageDropDown/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { LanguageIcon } from '@heroicons/react/24/outline';
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+import { useIntl } from 'react-intl';
+
+import { useLocale } from '@/hooks/useLocale';
+
+import styles from './index.module.css';
+
+const DropdownMenuDemo = () => {
+  const { availableLocales, currentLocale } = useLocale();
+  const intl = useIntl();
+
+  const ariaLabel = intl.formatMessage({
+    id: 'components.common.languageDropdown.label',
+  });
+
+  return (
+    <DropdownMenu.Root>
+      <DropdownMenu.Trigger asChild>
+        <button className={styles.iconWrapper} aria-label={ariaLabel}>
+          <LanguageIcon height="20" />
+        </button>
+      </DropdownMenu.Trigger>
+
+      <DropdownMenu.Portal>
+        <DropdownMenu.Content
+          align="start"
+          className={styles.dropDownContent}
+          sideOffset={5}
+        >
+          {availableLocales.map(({ name, code }) => (
+            <DropdownMenu.Item
+              key={name + code}
+              onClick={() => {}}
+              className={[
+                styles.dropDownItem,
+                code === currentLocale.code && styles.currentDropDown,
+              ].join(' ')}
+            >
+              <span className="text-sm">{name}</span>
```

Comment:
Since using span here and the default value of the span element display property is `inline`, this makes the height of the element higher than it should be. It would be more efficient to give the font properties of this element from a parent element and remove unnecessary span. Additionally, when I check Figma, can you add the `font-medium` class with font-weight set to 500